game-core/src/main/java/games/strategy/triplea/ui/panels/map/MapPanel.java

```diff
@@ -341,6 +342,18 @@ public GameData getData() {
     return gameData;
   }
 
+  public boolean getEditMode() {
+    final boolean isEditMode;
+    // use GameData from mapPanel since it will follow current history node
+    gameData.acquireReadLock();
+    try {
+      isEditMode = BaseEditDelegate.getEditMode(gameData);
```

Comment:
Just a small thing I noticed, there's no need to have an extra variable for this, this line can return the result directly.

Reply:
Agree, but for a follow-up. Didn't want to drastically update things too much to keep the review managable and make it clear this is moved code. The amount of cleanup that should be done in MapPanel and TripleAFrame goes really, really deep.